packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -74,6 +77,7 @@ class InteractiveViewer extends StatefulWidget {
     // use cases.
     this.maxScale = 2.5,
     this.minScale = 0.8,
+    this.scrollFrictionCoefficient = _kDefaultScrollFrictionCoefficient,
```

Comment:
This feels a bit misleading, InteractiveViewer is not a scrolling widget. If we use scrolling terms, users expect other scrolling conventions to apply, like having a ScrollPosition, and ScrollNotifications.

Reply:
It's a good point. How about we call it interactionEndFrictionCoefficient or dragEndFrictionCoefficient?

Reply:
That sounds good to me! :)